src/prop_types.js

```diff
@@ -82,6 +82,21 @@ export let VLayoutItemPropTypes = {
 
   // Used internally by VLayout
   _gutterTop: React.PropTypes.string,
-  _gutterBottom: React.PropTypes.string
+  _gutterBottom: React.PropTypes.string,
+
+  style: function (props, propName, componentName) {
+    const DANGEROUS_STYLES = [
+      "padding", "paddingTop", "paddingBottom", "paddingLeft", "paddingRight",
+      "margin", "marginTop", "marginBottom", "marginLeft", "marginRight",
+      "height", "width", "display", "position", "float",
```

Comment:
this is great! Could you also put this in `HLayoutItem`?

also, `VLayoutItems` can have a styled width, and `HLayoutItems` can have a styled height.